From 0dfac5b0077adfa8d9ac434538863a34a86df58d Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Fri, 1 Dec 2017 14:16:24 +0000
Subject: [PATCH] Add autogen.sh

Make it call buildconf for us.

The autogen.sh name is widely used in X.org, GNOME, and many other
projects.

See: https://github.com/cgwalters/build-api
---
 autogen.sh | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100755 autogen.sh

diff --git a/autogen.sh b/autogen.sh
new file mode 100755
index 0000000..6f1503e
--- /dev/null
+++ b/autogen.sh
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec ./buildconf
-- 
2.14.3

